packages/app/src/pages/translations/index.tsx

```diff
@@ -118,6 +123,8 @@ function TranslationsView(): JSX.Element {
       </Title>
 
       <Translations
+        canDelete={hasRequiredRights(userRights, RIGHTS.TRANSLATION_DELETE)}
+        canUpdate={hasRequiredRights(userRights, RIGHTS.TRANSLATION_UPDATE)}
```

Comment:
Why can't we put the rights-logic into the `Translations.tsx` file (`lib`) and use props here?